src/comp/pages/CreateProjectPage.tsx

```diff
@@ -26,36 +26,64 @@ class CreateProjectPage extends React.Component<
     };
   }
 
+  checkBlankSpace = (value: string) => {
+    let isError = true;
+    if (value.match(/\S/g)) {
+      isError = false;
+    }
+    return isError;
+  };
```

Comment:
これはただの関数としてクラス外においてください
また、これはそのまま `boolean` を返してください
```suggestion
  checkBlankSpace = (value: string) => {
    return !value.match(/\S/g);
  };
```